frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/HealthCard.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
```

Comment:
HealthCard should probably get renamed to StatusCard.tsx to align with other dashboards